tools/cldr-apps/js/src/views/CldrError.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div style="background: #ffffff; padding: 30px">
+    <a-card hoverable :title="subtypeString">
+      <template #extra
+        ><a-tag :title="status.subtype" :color="status.type.toLowerCase()">{{
+          status.type
+        }}</a-tag></template
+      >
+      <p>
+        {{ status.message }}
+      </p>
+      <template v-if="status.subtypeUrl" #actions
+        ><a class="warningReference" :href="status.subtypeUrl"
+          >How to Fix…</a
+        ></template
+      >
+    </a-card>
+  </div>
+</template>
+
+<script>
+// import * as cldrLoad from "../esm/cldrLoad.mjs";
+// import * as cldrStatus from "../esm/cldrStatus.mjs";
+export default {
+  computed: {
+    myClass() {
+      return `cldrError tr_${this.status.type} `;
+    },
+    subtypeString() {
+      return this.status.subtype
+        .split(/(?=[A-Z])/)
+        .map((s) => s)
+        .join(" ");
+    },
+  },
+  props: {
+    status: {
+      type: Object,
```

Comment:
is type an object, or a string? On the back end it's an enum, but in the json it looks very much like a string such as "Warning" -- or am I just totally confused?

Reply:
It's an object.

Reply:
I don't think we're talking about the same thing... `status` itself is an object, yes (I wrongly guessed CheckCLDR.CheckStatus but you say CheckStatusSummary and I believe you).

I was referring to `status.type`. On the back end that's `CheckCLDR.CheckStatus.Type`, which is an enum, and on the front end it's not an Object, it's a String. So, I think `type: Object` is wrong, it should be `type: String`